### ISSUE TITLE
Preprocess spectra: Linked x scales / cursor

### ISSUE THREAD

**Issue by stuart-cls** [Quasars/orange-spectroscopy#484]
Linking the x scales in the two `Spectra` widgets in Preprocess spectra would make it easier to compare spectral changes before and after processing.

Bonus points for linking the cursors as well.

**Comment by borondics** [comment]
The scales are not linked because there are operations (Cut) that can change the scale. This appears only in that particular preprocessor and the final view though.

I think linking the x value of the top-bottom cursor would be super nice too!

**Comment by stuart-cls** [comment]
Good point, I was thinking of the 2nd-derivative case mostly. There's so many different things that can happen in this widget :)
